### PR TITLE
added blitz and updated images

### PR DESCRIPTION
**Pull request by erondpowell**
Added Blitz article.
Swapped all "Users Roles and Permissions Photos" to have the newest branding.

CC @azrikahar

### CODE REVIEW

File: get-started-building-a-blitz-website-with-directus/index.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
+---
+title: Get Started Building a Blitz Website with Directus
+date: 2022-02-07
```
Comment (azrikahar) [inline get-started-building-a-blitz-website-with-directus/index.md:3]: May need to update this later on ([reference](https://github.com/directus/guides/commit/c842cc7d14becb465414ba1035699799c369ea06)), just a note 👍

File: get-started-building-a-blitz-website-with-directus/index.md [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+
+   ```
+   cd blitz
+   npm install
+   ```
+
+4. Start the development server.
+
+   ```
+   npm start
```
Comment (azrikahar) [inline get-started-building-a-blitz-website-with-directus/index.md:47]:
```suggestion
   npm run dev
```

**Review by azrikahar — APPROVED** [review]
LGTM!

**Comment by rijkvanzanten** [comment]
@erondpowell This ready to be merged once those two points above are addressed?

**Comment by erondpowell** [comment]
> @erondpowell This ready to be merged once those two points above are addressed?

Both points addressed.
Yes. Feel free to publish.

**Review by rijkvanzanten — APPROVED** [review]

**Pull request merged**